sig-etcd/vision.md

```diff
@@ -0,0 +1,100 @@
+# SIG etcd Vision
+
+The long-term success of the etcd project depends on the following:
+- Etcd is a reliable key-value storage
+- Etcd is simple to operate
+- Etcd is a standalone solution for managing infrastructure
+- Etcd scales beyond Kubernetes dimensions
+
+The goals and milestones listed here are for future releases. 
+The scope of release v3.6 has already been defined and is unlikely to change.
+
+## Etcd is a reliable key-value storage
+
+Reliability remains the most important property of etcd.
+The project cannot allow for another [data inconsistency incident].
+If we could only pick one thing from the list of goals above, this would be it.
+No matter what features we add in the future, 
+they must not diminish etcd's reliability. 
+We must establish processes and safeguards to prevent future incidents.
+
+How?
+- Etcd API guarantees are well understood, documented and tested.
+- Etcd adopts a production readiness review process for new features, similar to Kubernetes one.
+- Robustness tests should cover most of the API and most common failures.
+- New features must have accompanying e2e tests and be covered by robustness tests.
+- Etcd must be able to immediately detect corruption.
+- Etcd must be able to automatically recover from data corruption.
+ 
+[data inconsistency incident]: https://github.com/etcd-io/etcd/blob/main/Documentation/postmortems/v3.5-data-inconsistency.md
+
+## Etcd is simple to operate
+
+Etcd should be easy to operate.
+Currently, there are many steps involved in operating etcd,
+and some of these steps require external tools. 
+For example, Kubernetes provides tools to [downgrade/upgrade etcd].
+These tools are not part of the etcd,
+but they are available as part of the Kubernetes distribution of etcd.
+
+How?
+- Etcd should not require users to run periodic defrag
+- Etcd officially supports live upgrades and downgrades
+- Disaster recovery for Etcd & Kubernetes
+- Reliable cluster membership changes via learners with automated promotion
+- Two node etcd clusters
+
+## Etcd is a standalone solution for managing infrastructure
```

Comment:
(nit)
```suggestion
## Etcd is a standalone solution for managing infrastructure configuration
```

By itself, etcd is not really an infrastructure management tool. We can frame it as a place to store config, or as a key-value persistence layer that infrastructure management projects like to use.